contracts/CogitoGovernanceToken.sol

```diff
@@ -91,7 +91,7 @@ contract CogitoGovernanceToken is Context, AccessControl, ERC20Burnable, Pausabl
      * - The Cogito Governance Token should be 6 decimals instead of default decimals. This is only for display purpose.
      */
     function decimals() public view virtual override returns (uint8) {
-        return 6;
+        return 18;
```

Comment:
You need to delete function completely.
No need to over ride as the default decimals are 18.

Reply:
Fixed